src/components/EditorOperatorArgument.vue

```diff
@@ -4,8 +4,12 @@
       <el-input
         class="input-operator"
         :placeholder="argument.label"
-        :value="argument.value ? argument.value.toString() : ''"
-        @input="value => updateTemplateAndVariables(value)"
+        :value="
+          argument.value || typeof argument.value === 'boolean' ? argument.value.toString() : ''
+        "
```

Comment:
Could this be moved to a computed property?
```suggestion
        :value="argumentValue"
```